### PR TITLE
Improving styling of the sidebar slightly

### PR DESCRIPTION
**Pull request by dambem**
## Description of this PR
Just removing the github image, and putting in a big of a better colour

## Checklist
- [x] Tests all pass

### CODE REVIEW

File: ClimateMonitorV2/views/index.pug [hunk 1]
```diff
@@ -11,18 +11,17 @@ block content
...
 
         div(class="sidebar")
-            img(class="img-fluid", id="headerimage", src="sheffsense.png") 
             a(class="nav-link-active" href="/") Home
             a(class="nav-link" href="/about") About
             a(class="nav-link" href="/detailedstats") Detailed Statistics
             div(class="sidebar-footer")
                 a(href="https://github.com/dambem/ClimateMonitorV2" target="_blank")
-                    img(class="img-fluid", id="githubImage", src="github.png")
+                    img(class="img-fluid", id="githubImage", src="sheffsense.png")
```
Comment (M-Davies) [inline ClimateMonitorV2/views/index.pug]: Image is now `sheffsense.png` so the id across all the files should also be changed to reflect this

File: ClimateMonitorV2/views/about.pug [hunk 2]
```diff
@@ -12,13 +12,13 @@ block content
     body
 
         div(class="sidebar")
-            img(class="img-fluid", id="headerimage", src="sheffsense.png") 
-            a(class="nav-link" href="/") Home
-            a(class="nav-link-active" href="/about") About
+            //img(class="img-fluid", id="headerimage", src="sheffsense.png")
```
Comment (M-Davies) [inline ClimateMonitorV2/views/about.pug]: We can probably remove the commented out stuff

File: ClimateMonitorV2/views/about.pug [hunk 3]
```diff
@@ -12,13 +12,13 @@ block content
     body
 
         div(class="sidebar")
-            img(class="img-fluid", id="headerimage", src="sheffsense.png") 
-            a(class="nav-link" href="/") Home
-            a(class="nav-link-active" href="/about") About
+            //img(class="img-fluid", id="headerimage", src="sheffsense.png") 
+            a(class="nav-link-active" href="/") Home
```
Comment (M-Davies) [inline ClimateMonitorV2/views/about.pug]: The current page title in the sidebar is no longer highlighted when I change pages. See [main.css](https://github.com/dambem/ClimateMonitorV2/blob/master/ClimateMonitorV2/public/stylesheets/main.css#L20-L27) for how I implemented it

File: ClimateMonitorV2/views/about.pug [hunk 4]
```diff
@@ -12,13 +12,13 @@ block content
...
+            a(class="nav-link" href="/about") About
             a(class="nav-link" href="/detailedstats") Detailed Statistics
             div(class="sidebar-footer")
                 a(href="https://github.com/dambem/ClimateMonitorV2" target="_blank")
-                    img(class="img-fluid", id="githubImage", src="github.png")
+                    img(class="img-fluid", id="githubImage", src="sheffsense.png")
```
Comment (M-Davies) [inline ClimateMonitorV2/views/about.pug]: Img is a little too close to the bottom of the sidebar on all pages. For me, the bottom of the logo is now off the screen

File: ClimateMonitorV2/public/stylesheets/main.css [hunk 5]
```diff
@@ -28,14 +28,16 @@ body {
 
 /* Sidebar setup */
 .sidebar {
```
Comment (M-Davies) [inline ClimateMonitorV2/public/stylesheets/main.css:29]: Could we make the sidebar page names bold? I find it a little hard to read in the current format (Using [font-weight](https://www.w3schools.com/cssref/pr_font_weight.asp) will probably work well)

**Review by M-Davies — CHANGES_REQUESTED** [review]
Sorry for the long review @dambem 😬. I promise it's reasonable advice

**Comment by dambem** [comment]
All issues should be fixed now, have redone the design slightly

File: ClimateMonitorV2/views/index.pug [hunk 6]
```diff
@@ -10,19 +10,21 @@ block content
...
-            img(class="img-fluid", id="headerimage", src="sheffsense.png") 
+            div(class="sidebar-header")
+                a(href="https://github.com/dambem/ClimateMonitorV2", id="sheffsenseImageHolder", target="_blank")
+                    img(class="img-fluid", id="sheffsenseImage", src="sheffsense.png")
+
             a(class="nav-link-active" href="/") Home
             a(class="nav-link" href="/about") About
             a(class="nav-link" href="/detailedstats") Detailed Statistics
             div(class="sidebar-footer")
                 a(href="https://github.com/dambem/ClimateMonitorV2" target="_blank")
```
Comment (M-Davies) [inline ClimateMonitorV2/views/index.pug]: The href of the image points to the github (same as the sheffsense logo). We shouldn't need two images in the sidebar pointing links to the same location

**Review by M-Davies — CHANGES_REQUESTED** [review]
One last small nitpick

File: ClimateMonitorV2/views/index.pug (hunk 6 above)
Comment (dambem) [reply ClimateMonitorV2/views/index.pug]: Fixed in latest commit now

**Review by M-Davies — APPROVED** [review]

**Pull request merged**